scene/3d/skeleton.cpp

```diff
@@ -691,6 +693,8 @@ void Skeleton::physical_bones_remove_collision_exception(RID p_exception) {
 	_physical_bones_add_remove_collision_exception(false, this, p_exception);
 }
 
+#endif _3D_DISABLED
```

Comment:
I'm not sure `#endif` can take a named define? Usually we put `#endif // _3D_DISABLED`